internal/pkg/runtime/engines/imgbuild/rpc/args.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2018, Sylabs Inc. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) 2018-2019, Sylabs Inc. All rights reserved.
```

Or

```suggestion
// Copyright (c) 2019, Sylabs Inc. All rights reserved.
```

Reply:
Do range, not changing it to a single year. Code exists in the file that was created from 2018-2019 yes? Per file the Copyright year should reflect the years that is inclusive of any code written.

Reply:
Agree, but then should it be `2017-2019`?

Reply:
Only if that file existed in 2017

Reply:
> The Copyright header in a file is per file, not project life.

Ahh, that makes sense.

Reply:
Actually this file looks brand new... (just created in this PR...)

Reply:
All the new files should have 2019 for the Copyright year then.

Reply:
Okay, so my second suggestion would be correct. Thanks for clarifying this :+1: